execution/evm/stack.go

```diff
@@ -21,23 +21,32 @@ import (
 
 	. "github.com/hyperledger/burrow/binary"
 	"github.com/hyperledger/burrow/execution/errors"
+	"math"
 )
 
+// Change the length of this zero array to tweak the size of the block of zeros
+// written to the backing slice at a time when it is grown. A larger number may
+// lead to fewer calls to append to achieve the desired capacity although it is
+// unlikely to make a lot of difference.
+var zeroWords []Word256 = make([]Word256, 32)
```

Comment:
This ends up being 1 MiB which is quite a bit 'minimum chunk'. In memory.go we are just using 32 bytes. So `var zeroWord Word256` would be fine here. In fact we actually already have a constant you can use for this: https://github.com/hyperledger/burrow/blob/develop/binary/word256.go#L27. May as well import that and save the allocation.